### ISSUE TITLE
secret-job-scripts: compare_variables function conditionally tied to poolsUpdate value instead of variablesUpdate

### ISSUE THREAD

**Issue by bobpace** [airflow-helm/charts#139]
**What is the bug?**

There appears to be a typo here, which at the moment requires the `poolsUpdate` value to be set to define the `compare_variables` function when it seems that `variablesUpdate` was what was intended.

https://github.com/airflow-helm/charts/blob/8611c766992a9f7156aad450f17beb2213cf667c/charts/airflow/templates/jobs/secret-job-scripts.yaml#L242

Setting variablesUpdate to true and leaving poolsUpdate false will result in this error trying to set variables:
```
Traceback (most recent call last):
  File "/job_scripts/create_variables.py", line 36, in <module>
    if compare_variables(v, query_result):
NameError: name 'compare_variables' is not defined
Sentry is attempting to send 1 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
```

**What are your Helm values?**

```
  variablesUpdate: true
  poolsUpdate: false
```

**What is your Kubernetes Version?**:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-14T05:15:04Z", GoVersion:"go1.15.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:41:49Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
$ helm version
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
```

**Comment by thesuperzapper** [comment]
@bobpace good catch

**Comment by thesuperzapper** [comment]
@bobpace this has been fixed in `8.0.7`
